test/Autofac.Configuration.Test/Autofac.Configuration.Test.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net461;netcoreapp2.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net472;netcoreapp3.0</TargetFrameworks>
```

Comment:
I don't think we need to keep the net472 project tests around, do we? I think this is an artifact of DNX and things not working; and we don't have a desktop .NET target in Autofac core anymore. (I mean, we still exclude `*.xproj` down on line 19, right? Love that DNX.)